### ISSUE TITLE
Hanging tread remains in vm_handler depend tests

### ISSUE THREAD

**Issue by simonOpenV** [openvstorage/integrationtests#622]
```
2018-02-21 03:56:39 15300 +0100 - DEV-4N-199-191 - 14012/140619793045248 - scenario_helpers-vm_handler/vm_handler.py - prepare_vm_disks - 15858 - INFO - Prepped everything for VM mds-regression-000.
2018-02-21 03:56:39 15400 +0100 - DEV-4N-199-191 - 14012/140619793045248 - scenario_helpers-vm_handler/vm_handler.py - create_vms - 15859 - INFO - Creating vms
2018-02-21 03:56:39 15400 +0100 - DEV-4N-199-191 - 14012/140619793045248 - scenario_helpers-vm_handler/vm_handler.py - create_vms - 15860 - INFO - Initializing creation of vm mds-regression-000
2018-02-21 03:56:39 15400 +0100 - DEV-4N-199-191 - 14012/140619793045248 - scenario_helpers-vm_handler/vm_handler.py - create_vm - 15861 - INFO - Creating VM `mds-regression-000`
2018-02-21 03:56:39 38900 +0100 - DEV-4N-199-191 - 14012/140616490809088 - scenario_helpers-vm_handler/vm_handler.py - listener - 15863 - DEBUG - Connection message received: ['20c5885c-228a-4d68-bf4a-5ad1861e7312_mds-regression-000']
2018-02-21 03:56:42 46500 +0100 - DEV-4N-199-191 - 14012/140619793045248 - scenario_helpers-vm_handler/vm_handler.py - create_vm - 15869 - INFO - Created VM `mds-regression-000`!
```

Where it keeps on hanging.
This has been introduced due to recent socket changes to fix #586

**Comment by simonOpenV** [comment]
Will be fixed by #621

**Comment by JeffreyDevloo** [comment]
Released in https://github.com/openvstorage/integrationtests/releases/tag/3.4.1
Packaged in openvstorage-test_3.4.1-1_amd64.deb
